SmellDetector/SmellModel/CaDETClassDTO.cs

```diff
@@ -1,16 +1,22 @@
-﻿using System.Collections.Generic;
-using SmellDetector.Controllers;
+﻿using System;
+using System.Collections.Generic;
 
 namespace SmellDetector.SmellModel
 {
     public class CaDETClassDTO
     {
-        public Dictionary<string, MetricsDTO> CodeItemMetrics { get; set; }
+        public string FullName { get; set; }
+        public Dictionary<string, double> CodeItemMetrics { get; set; }
```

Comment:
CodeItem is a deprecated term we used last year. CodeSnippet is what we use now, so refactor all occurences.